src/ontology/mondo-edit.obo

```diff
@@ -407448,9 +407448,10 @@ xref: MESH:D011248 {source="MONDO:equivalentTo"}
 xref: NCIT:C35169 {source="MONDO:equivalentTo"}
 xref: SCTID:173300003 {source="MONDO:equivalentTo"}
 xref: UMLS:C0151864 {source="MONDO:notFoundInDiseaseSubset", source="NCIT:C35169"}
-is_a: MONDO:0000001 {source="https://github.com/monarch-initiative/mondo/issues/652"} ! disease or disorder
+is_a: MONDO:0700003 ! obstetric disorder
```

Comment:
could you please add a source annotation here
you can add the ticket https://github.com/monarch-initiative/mondo/issues/3161 and your ORCID